tests/Resources/assert.test.js

```diff
@@ -1215,59 +1215,59 @@ describe('assert', function () {
 
 	describe('.strict', () => {
 		it('is a function', function () {
-			assert.strict.should.be.a.Function(); // it's an alias for assert.ok
+			should(assert.strict).be.a.Function(); // it's an alias for assert.ok
 		});
 
 		it('is available off of itself', function () {
-			assert.strict.strict.should.be.ok();
+			should(assert.strict.strict).be.ok();
 		});
 
 		describe('#ok', () => {
-			it('is a function', () => assert.strict.ok.should.be.a.Function());
+			it('is a function', () => should(assert.strict.ok).be.a.Function());
 		});
 
 		describe('#ifError', () => {
-			it('is a function', () => assert.strict.ifError.should.be.a.Function());
+			it('is a function', () => should(assert.strict.ifError).be.a.Function());
 		});
 
 		describe('#fail', () => {
-			it('is a function', () => assert.strict.fail.should.be.a.Function());
+			it('is a function', () => should(assert.strict.fail).be.a.Function());
 		});
 
 		describe('#deepEqual', () => {
-			it('is a function', () => assert.strict.deepEqual.should.be.a.Function());
+			it('is a function', () => should(assert.strict.deepEqual).be.a.Function());
 		});
 
 		describe('#throws', () => {
-			it('is a function', () => assert.strict.throws.should.be.a.Function());
+			it('is a function', () => should(assert.strict.throws).be.a.Function());
 		});
 
 		describe('#doesNotThrow', () => {
-			it('is a function', () => assert.strict.doesNotThrow.should.be.a.Function());
+			it('is a function', () => should(assert.strict.doesNotThrow).be.a.Function());
 		});
 
 		describe('#rejects', () => {
-			it('is a function', () => assert.strict.rejects.should.be.a.Function());
+			it('is a function', () => should(assert.strict.rejects).be.a.Function());
 		});
 
 		describe('#doesNotReject', () => {
-			it('is a function', () => assert.strict.doesNotReject.should.be.a.Function());
+			it('is a function', () => should(assert.strict.doesNotReject).be.a.Function());
 		});
 
 		describe('#strictEqual', () => {
-			it('is a function', () => assert.strict.strictEqual.should.be.a.Function());
+			it('is a function', () => should(assert.strict.strictEqual).be.a.Function());
 		});
 
 		describe('#notStrictEqual', () => {
-			it('is a function', () => assert.strict.notStrictEqual.should.be.a.Function());
+			it('is a function', () => should(assert.strict.notStrictEqual).be.a.Function());
 		});
 
 		describe('#deepStrictEqual', () => {
-			it('is a function', () => assert.strict.deepStrictEqual.should.be.a.Function());
+			it('is a function', () => should(assert.strict.deepStrictEqual).be.a.Function());
 		});
-
+t
```

Comment:
<!--
  3 failure:  tests/Resources/a..., tests/Resources/a..., tests/Resources/a...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: tests/Resources/assert.test.js;
  Line: 1268;
-->
- :no_entry_sign: tests/Resources/assert.test.js line 1268 – 't' is not defined. (no-undef)
- :no_entry_sign: tests/Resources/assert.test.js line 1268 – Expected indentation of 2 tabs but found 0. (indent)
- :no_entry_sign: tests/Resources/assert.test.js line 1268 – Missing semicolon. (semi)